.github/stale.yml

```diff
@@ -0,0 +1,47 @@
+daysUntilStale: 300
+daysUntilClose: 30
+exemptLabels:
```

Comment:
Why this?

Reply:
My original intent was to help exclude issues from closing. But instead I have added the only: pulls key so it won't touch issues. I think with this we can remove the exempt labels. My original intent was to avoid closing important issues.

Reply:
I agree. `only: pulls` makes this unnecessary.